public/style.css

```diff
@@ -196,4 +196,8 @@
 
```

Comment:
I noticed that the character height of the line numbers is slighty off from the text lines. (1 px).

Not super important to fix, but maybe we can check quickly:
![image](https://user-images.githubusercontent.com/1754968/210562618-b6b3414b-a03a-4467-a718-0d1b50960f1e.png)

Reply:
Ahh interesting, this looks like a css line-height issue, I'll have a look in the dom inspector to see if I can track down the cause within neodash